### DISCUSSION TITLE
更新订阅疑问

### DISCUSSION THREAD

**Discussion by iwalke** [clash-verge-rev/clash-verge-rev#1533] [General]
![image](https://github.com/user-attachments/assets/3b7fbd73-7efa-49f7-8ed1-db120be70e6d)

这个内核代理更新跟系统代理更新有什么区别

**Comment by minusmulticoloredper** [comment]
使用系统代理更新：使用已经设置的系统代理更新配置文件，如果要经过Clash需要将系统代理设置为Clash的代理地址。
使用内核代理更新：使用Clash内核提供的代理更新配置文件，经过Clash，即使没有设置系统代理。

**Reply by iwalke** [reply]
言下之意，cvr 还支持另外一个代理启动么？

如果只开了一个 cvr，那使用系统代理更新和使用内核代理更新，是不是就没区别了？都经过 cvr，怎么连接到订阅 由cvr定义的规则定？

**Reply by minusmulticoloredper** [reply]
系统代理指的是操作系统中的代理服务器设置，只支持http和socks5协议，有代理服务器的ip地址和端口，设置后请求就会经过代理服务器。
而Clash内核会在本地监听一个端口，此时将代理服务器的ip设置为本地回环地址，端口填写Clash监听的端口，那么请求就会经过Clash内核。
Clash verge rev中的系统代理模式，其实是自动把操作系统中的代理服务器的ip和端口设置为本地回环地址和Clash监听端口。

**Reply by kty00** [reply]
> 使用系统代理更新：使用已经设置的系统代理更新配置文件，如果要经过Clash需要将系统代理设置为Clash的代理地址。 使用内核代理更新：使用Clash内核提供的代理更新配置文件，经过Clash，即使没有设置系统代理。

这个代理更新默认走全局模式么？
